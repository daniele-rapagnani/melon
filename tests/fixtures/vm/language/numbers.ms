io.print(string.format("%.5f", 5e8 + 5e-5));
io.print(5.456e3);