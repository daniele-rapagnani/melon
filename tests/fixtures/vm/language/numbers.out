500000000.00005
5456.000000000
